lib/expand.js

```diff
@@ -420,6 +420,9 @@ async function _expandObject({
   const nests = [];
   let unexpandedValue;
 
+  // Add expansion map to the processing options
+  options = {...options, expansionMap};
```

Comment:
Do we need to pass `expansionMap` from `expand` method to `_expandValue` options, which is then passed to `_expandIri` in downstream?

Reply:
Yes, `_expandIri` should always receive the `expansionMap` if it was passed in -- otherwise we're going to miss corner cases.